docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,483 @@ Review the following information about the {kib} 8.3.0 release.
+=== Features
+
+{kib} 8.3.0 adds the following new and notable features.
+
+Alerting::
+* Adds circuit breaker for max number of actions by connector type {kibana-pull}128319[#128319]
+* Adds `bulkEdit` method to alerting rulesClient and internal _bulk_edit API, that allow bulk editing of rules {kibana-pull}126904[#126904]
+
+Cases::
+* Adds average time to close metric in Cases {kibana-pull}131909[#131909]
+* View all alerts attached to a case in the alerts table. The feature is experimental {kibana-pull}131883[#131883]
+* Adds severity field to Cases {kibana-pull}131626[#131626]
+* Adds the ability to delete comments in Cases {kibana-pull}130254[#130254]
+
+Dashboard::
+Enables the new controls by default {kibana-pull}131341[#131341]
+
+Discover::
+* Adds the ability to edit dataView, query, & filters {kibana-pull}131688[#131688]
+* Extend Elasticsearch query rule with search source based data fetching {kibana-pull}124534[#124534]
+
+Elastic Security::
+For the Elastic Security 8.3.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+Changes to agent upgrade modal to allow for rolling upgrades {kibana-pull}132421[#132421]
+
+Lens & Visualizations::
+* Adds method to re-link visualizations with missing `SavedSearch` {kibana-pull}132729[#132729]
+* Adds support of Data View switching for Agg-Based visualizations {kibana-pull}132184[#132184]
+* Adds `showTooltip` argument {kibana-pull}130530[#130530]
+* Adds `orderBucketsBySum` {kibana-pull}130457[#130457]
+* Adds lines visibility for lines chart {kibana-pull}130452[#130452]
+* Adds `pointsRadius`, `showPoints`, and `lineWidth` {kibana-pull}130391[#130391]
+* Adds mark size configuration {kibana-pull}130361[#130361]
+* Adds `xyVis` split chart {kibana-pull}130229[#130229]
+
+Machine Learning::
+* Adds the ability to create anomaly detection jobs from Lens visualizations {kibana-pull}129762[#129762]
+* Additional testing of trained model in UI {kibana-pull}129209[#129209]
+
+Management::
+* Adds saved object relationships to data view management {kibana-pull}132385[#132385]
+* Adds support for feature_states {kibana-pull}131310[#131310]
+
+Monitoring::
+Adds the Stack monitoring health API {kibana-pull}132705[#132705]
+
+Observability::
+* Adds the ability to bulk attach multiple alerts to a Case {kibana-pull}130958[#130958]
+* Adds rule details page {kibana-pull}130330[#130330]
+* Adds span link {kibana-pull}126630[#126630]
+* Adds ML expected model bounds as an option to Comparison controls {kibana-pull}132456[#132456]
+
+Platform::
+Adds `xyVis` and `layeredXyVis` {kibana-pull}128255[#128255]
+
+Querying & Filtering::
+Improves the current filter/search experience {kibana-pull}128401[#128401]
+
+Security::
+* Adds a manual refresh button {kibana-pull}132766[#132766]
+* Adds the metadata tab {kibana-pull}131465[#131465]
+* Adds the ability to use {kib} feature privileges with API keys {kibana-pull}130918[#130918]
```

Comment:
(we decided to not include in this in the release notes, it's too internal - I removed the release note label from PR).
```suggestion
```